modules/hitl2/src/views/full/Components/Comment.tsx

```diff
@@ -1,26 +1,35 @@
-import React, { FC, useContext } from 'react'
+import { ContentSection, lang } from 'botpress/shared'
 import moment from 'moment'
+import React, { FC, useContext } from 'react'
 
+import { CommentType } from '../../../types'
 import { Context } from '../app/Store'
+import style from '../style.scss'
 
-import { CommentType } from '../../../types'
+interface Props {
+  threadId: string
+  comment: CommentType
+}
 
-const Comment: FC<CommentType> = props => {
+const Comment: FC<Props> = ({ comment, threadId }) => {
+  const { agentId, content, createdAt } = comment
   const { state } = useContext(Context)
 
   function formatDate(str) {
     return moment(str).format('DD/MM/YYYY')
   }
 
-  const agent = state.agents[props.agentId]
+  const agent = state.agents[agentId] || { fullName: 'test name', id: '1' }
 
   return (
-    <div style={{ marginBottom: 'var(--spacing-xx-large)' }}>
-      <p>{props.content}</p>
-      <p className="bp3-text-small bp3-text-muted">
-        {formatDate(props.createdAt)} <span>⋅</span> {agent.fullName || agent.id}
+    <ContentSection title={`#${threadId}`}>
```

Comment:
I'm not sure but those uds can can become quite ugly, we might want to display a substring like the 1st 6 chars or something

Reply:
Isn't it supposed to be the conversation number? seemed to be incremental in the design on purpose

Reply:
In the design yes but for real it isn't, I'm telling you